### ISSUE TITLE
Integration of OpenGotha tournament functionality with openstudyroom.org's tournament tools.  

### ISSUE THREAD

**Issue by alexmitrani** [climu/openstudyroom#311]
OpenGotha is programmed in Java and is also on github.  http://vannier.info/jeux/gotournaments/opengotha.htm

It is used by many Go tournament organisers worldwide.  

To the extent that openstudyroom.org gets more use for tournaments by Go clubs and associations, it would be useful to integrate OpenGotha functionality with openstudyroom.org's web interface.  This would enable tournaments to be completely organised on openstudyroom.org, and would probably lead to openstudyroo..org being used even more by more Go communities.

**Comment by climu** [comment]
Good idea.

For the record Opengotha is [licenced under PGL](https://github.com/lucvannier/opengotha/issues/9).

I will need help there.

**Comment by alexmitrani** [comment]
Yes, and ideally Swiss tournaments too (OpenGotha supports both systems).  The idea would be to make tournament easier and more transparent.  As it is, organising a tournament involves using several different systems, often on several devices, with many imports, exports and transfers of data between them:
- google forms (transfer to OpenGotha)
- OpenGotha (publish outputs to HTML, transfer from computer to smartphone)
- whatsapp (share pairings and standings during tournament)
- website and social networks (share final standings and tournament report).  

I hope this helps explain why I am wishfully thinking of a simpler, more integrated workflow.  

Thanks for giving this thought.

**Comment by raylu** [comment]
integrating a GUI app written in a different language with a webapp is an insane amount of work. I wouldn't count on this happening anytime soon... or ever.

it is almost certainly less work to rewrite a tournament management tool from scratch on top of OSR... or just make a separate webapp entirely for it

**Comment by alexmitrani** [comment]
A more modest but still useful improvement would be to enable export / import of information between OpenGotha and OSR tournament tools.  OpenGotha stores all tournament information in XML format.  OpenGotha allows importing information, such as lists of players, from several formats and existing databases.  OSR tournament pages would ideally be able to import pairings and standings from OpenGotha.

**Comment by fakirsu** [comment]
Hello, 
For information, it seems that a new version of OpenGotha is preparing. 
The developpers launched a call very recently (June 2023) to report any bugs and complaints. 
I mention it here if it can help. It is true that the integration of opengotha and OSR would be great.
